SchedulearnBackend/SchedulearnBackend/UserFriendlyExceptions/UnauthorisedException.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace SchedulearnBackend.UserFriendlyExceptions
+{
+    public class UnauthorisedException : UserFriendlyException
+    {
+        public UnauthorisedException() : this(null)
+        {
+        }
+
+        public UnauthorisedException(string message) : this(message, null)
+        {
+        }
+
+        public UnauthorisedException(string message, Exception inner) : base(HttpStatusCode.Unauthorized, message, inner)
```

Comment:
I recommend using the word "Unauthorized" instead of "Unauthorised", since the attributes above controllers are also written with a Z. Both are correct, but might as well keep it consistent.